gradle/libs.versions.toml

```diff
@@ -0,0 +1,88 @@
+[versions]
+compileSdkVersion = "31"
+exampleCompileSdk = "33"
+minSdk = "24"
+java = "1.8"
+buildTools = "30.0.3"
+
+core = "2.3.0"
```

Comment:
nit: `paykit`? `core` feels too generic, it's not clear what it refers to. Even better would be to just inline this version into the dependency declaration, usually there's no reason to declare version separately when it's only used by one dep:

```toml
[libraries]
paykit = "app.cash.paykit:core:2.3.0"
```